### DISCUSSION TITLE
[Other] How to upload files?

### DISCUSSION THREAD

**Discussion by heygambo** [jonaswinkler/paperless-ng#1055] [Support]
I have paperless running but I'm wondering how to upload files.
I'm logged in as the superadmin but I don't see any upload button.

The docs I have found so far don't explain how to upload files. I might be blind :)

**Comment by jonaswinkler** [comment]
It's on the right hand side of the dashboard.

![image](https://user-images.githubusercontent.com/17569239/119332427-dd576b00-bc88-11eb-8a32-ecbc65a8f565.png)

**Reply by heygambo** [reply]
I don't know how I could have missed that the whole time. I was expecting that on the documents page somehow.
